packages/core/src/util/applicationUnderMonitoring.js

```diff
@@ -166,18 +168,7 @@ function getMainPackageJsonPathStartingAtDirectory(startDirectory, cb) {
       if (
         // @ts-ignore
         (process._preload_modules && process._preload_modules.length > 0) ||
-        (process.env.NODE_OPTIONS &&
-          (process.env.NODE_OPTIONS.indexOf('--experimental-loader') !== -1 ||
-            process.env.NODE_OPTIONS.indexOf('--import') !== -1)) ||
-        (process.execArgv &&
-          process.execArgv.length > 0 &&
-          ((process.execArgv[0].indexOf('--experimental-loader') !== -1 &&
-            process.execArgv[0].indexOf('esm-loader.mjs')) !== -1 ||
-            (process.execArgv[0].indexOf('--experimental-loader') !== -1 &&
-              process.execArgv[1].indexOf('esm-loader.mjs') !== -1) ||
-            (process.execArgv[0].indexOf('--import') !== -1 && process.execArgv[0].indexOf('esm-register.mjs')) !==
-              -1 ||
-            (process.execArgv[0].indexOf('--import') !== -1 && process.execArgv[1].indexOf('esm-register.mjs') !== -1)))
+        isESMApp()
```

Comment:
q: the utility is called "esmSupportedVersion". How about we name it "util/esm.js"?

> esmUtils = require(util/esm)
> esmUtils.isESMApp()